app/views/submissions/new.html.erb

```diff
@@ -0,0 +1,41 @@
+<div class="show_and_edit">
+  <%= form_for new_post do |post| %>
+    <% if new_post.errors[:base].present? %>
```

Comment:
the `show_errors` method is already doing this presence check, right?  This seems redundant. Could you move the `div` to the helper method?

Reply:
"Could you move the div to the helper method?" That's a good idea. The reason I used the `if` statement was because I was getting styled divs in the view when the form was first displayed. If the helper holds the div, then I'd get around that issue.